### DISCUSSION TITLE
loss_weights decay with the trainning process?

### DISCUSSION THREAD

**Discussion by Xhao12138** [lululxvi/deepxde#1700] [Q&A]
I want  to achieve the loss weight as the training process goes down.This is my code:
model.complie('adam', lr=0.001, loss_weights=[1,10000], external_trainable_variables=C)
I hope 10000 to linearly decay to 1 when the trainning is over.
Thanks for your help!

**Comment by lululxvi** [comment]
There is no decay in DeepXDE.
